docs/stdlib/select.md

```diff
@@ -66,3 +66,35 @@ Options      |         Description           | Required?
 
 Only the point with the exact median value for the `field` name
 provided will be forwarded.
+
+## top - sub
+
+```
+... | select.top -n number -by field | ...
+```
+
+Options  |                              Description                              | Required?
+-------- | --------------------------------------------------------------------- | --------- :
+`n`      | how many of the fields with the highest values of `by` to keep        | Yes
+`by`     | name of the field to sort by before figuring out the top `n` to keep  | Yes
+
+Let the `n` points that have the highest value of `by` pass through. 
+
+**Note** since we use sort to sort by the `by` field we end up not emitting
```

Comment:
Double negation here? We emit points without time field or we don't do so? Maybe just say that we drop the time field from the forwarded points (regrettably so...this sort-drops-time business has tripped me up repeatedly, especially when using subs where I forget sort is in there, and then nothing works downstream. @dmajda can we revisit this?)